core/tauri/src/scope/fs.rs

```diff
@@ -79,11 +79,33 @@ fn push_pattern<P: AsRef<Path>, F: Fn(&str) -> Result<Pattern, glob::PatternErro
 ) -> crate::Result<()> {
   let path: PathBuf = pattern.as_ref().components().collect();
   list.insert(f(&path.to_string_lossy())?);
-  #[cfg(windows)]
+  #[cfg(any(windows, target_os = "android"))]
```

Comment:
I don't think windows needs all this extra logic, can we just leave this logic for android only and keep windows logic separate?

Reply:
actually all targets needs this, because we always canonicalize() the path when checking it on is_allowed and if the path has symlinks we need to resolve that here too (unless we find a better way to check paths)